cherrypy/test/test_static.py

```diff
@@ -193,6 +209,19 @@ def test_static(self):
         #   we just check the content
         self.assertMatchesBody('^Dummy stylesheet')
 
+        # Check a filename with utf-8 characters in it
+        ascii_fn = 'has_utf-8_character_.html'
+        url_quote_fn = 'has_utf-8_character_%E2%98%83.html'
+        cd = '''attachment; filename="%s"; filename*=UTF-8\'\'%s'''
```

Comment:
no need to have a multiline literal here:
```suggestion
        cd = 'attachment; filename="%s"; filename*=UTF-8\'\'%s'
```